docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -465,3 +467,118 @@ The following overrides are available:
   For more information, see [Helm chart: Prerequisites](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#prerequisites).
 
 - Overrides to use a secret for LDAP authentication. Refer to [Create secrets for Kubernetes](../../../../secure/authentication/ldap-authentication-ysql/#create-secrets-for-kubernetes).
+
+## Configure Multi-Cluster Services
```

Comment:
Configure the provider for multi-cluster environments?

Reply:
Made it "Configure Kubernetes multi-cluster environment", because using "provider" becomes confusing, as we have one more section with this title: "Configure the cloud provider for MCS"